packages/core/test/hardhat/data-verification-mechanism/VotingV2.js

```diff
@@ -5188,6 +5188,47 @@ describe("VotingV2", function () {
       if (toBN(slashedTokens).isNeg()) assert(startingAccountBalances[voter].gte(toBN(slashedTokens).abs()));
     }
   });
+  it("Changing max rolls while requests are unresolved does not impact requests", async function () {
+    // Consider if requests are resolveable but before they can be resolved the maxRolls variable is changed. If this
+    // was to happen then it is possible that requests that could be resolved get deleted before they resolve. To
```

Comment:
> requests that could be resolved get deleted before they resolve

I believe we are testing something different that was not a problem prior to the changes. A `maxRoll` update wouldn't have caused a resolvable request to be deleted because[ is resolvable is evaluated first](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol#L1025).  However, before your fix, a non-voted rolling request could be deleted because of the retroactive behaviour of `setMaxRoll` 

I think in this test we should test the following:
- maxRolls = 2 by default
- create a price request
- moveToNextRound x3
- processResolvablePriceRequests would delete this request but instead we try to setMaxRolls(4) 
- check that the price request has been deleted verifying that the setMaxRolls is non-retroactive

Reply:
@Reinis-FRP added this :) please let us know if it tracks with what you were expecting.